released/packages/coq-math-classes/coq-math-classes.8.9.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "b.a.w.spitters@gmail.com"
+
+homepage: "https://github.com/coq-community/math-classes"
+dev-repo: "git+https://github.com/coq-community/math-classes.git"
+bug-reports: "https://github.com/coq-community/math-classes/issues"
+license: "Public Domain"
+
+synopsis: ""
+description: """
+A library of abstract interfaces for mathematical structures in Coq.
+"""
+
+build: [
+  [ "./configure.sh" ]
+  [ make "-j%{jobs}%" ]
+]
+install: [make "install"]
+depends: [
+  "ocaml"
+  "coq" {(>= "8.6" & < "8.11~")}
+  "coq-bignums" 
+]
+
+tags: [
+  "logpath:MathClasses"
```

Comment:
Please add `"date:YYYY-MM-DD"` here for when the release was made on GitHub. Preferably, also add some keywords and a category, but this is optional.